test/test_client.py

```diff
@@ -133,7 +133,16 @@ async def test_async_client_rest(aioresponses, async_client):
 
     aioresponses.delete(url, status=200)
     await client.delete("/octocat")
-    aioresponses.assert_called_with(url, "DELETE", data="null")
+    aioresponses.assert_called_with(
+        url,
+        "DELETE",
+        headers={
+            "Accept": "application/vnd.github+json",
+            "Authorization": "Bearer abc",
+        },
```

Comment:
I'm not sure what's going on here. At first I was able to reproduce this locally, and then I did something and now I can't anymore, but it still reproduces in CI?

I guess I could edit the test to be less strict about what it asserts, but I'm worried this might be uncovering an actual issue :/

Reply:
Do you mind adding something in the commit message about this test change?

Reply:
Will do.

Turns out that locally I had somehow managed to re-install the old dependency set. So it's at least consistent. I guess it's a change to aiohttp.